inputs/BaseInput.php

```diff
@@ -13,6 +13,18 @@
  */
 abstract class BaseInput
 {
+    /**
+     * Add options
```

Comment:
Remember the first line in a source code documentation block should be a summary sentence of the method that is finished with a `.`.
The next sentence is a good summary for the method, so just remove this line.